controllers/admin/AdminImportController.php

```diff
@@ -1473,11 +1953,21 @@ protected function categoryImportOne($info, $default_language_id, $id_lang, $for
         }
     }
 
-    public function productImport($offset = false, $limit = false, &$crossStepsVariables = false, $validateOnly = false, $moreStep = 0)
-    {
+    public function productImport(
+        $offset = false,
+        $limit = false,
+        &$crossStepsVariables = false,
+        $validateOnly = false,
+        $moreStep = 0
+    ) {
+        if (!is_array($crossStepsVariables)) {
+            $crossStepsVariables = array();
```

Comment:
Only variables can be passed by reference. It's impossible to fall back on the default value unless a `null` value variable is passed. Core and native modules don't do that. And there is no way such a case can happen somewhere else. We figured this out with Aurélien (@toutantic).
`$crossStepsVariables` is ment to be an array, and any other value would just be a mishap.